### ISSUE TITLE
Full paths instead of relative paths

### ISSUE THREAD

**Issue by wlandau** [richfitz/storr#50]

**Comment by wlandau** [comment]
```r
library(storr)
x = storr_rds("storr")
x$driver$path
## [1] "storr"
x$set("a", 1)
x$list()
## [1] "a"
dir.create("subdir")
setwd("subdir")
x$list()
## character(0) # Should be "a"
```

Implementation could get a tad tricky: what if the user supplies the full path to `storr_rds()` in the first place?

A fix here would fix wlandau-lilly/drake#13. Related: r-lib/memoise#51.

**Comment by richfitz** [comment]
Thanks - as noted on your PR I agree with you entirely
